test/test_interface.cc

```diff
@@ -135,6 +135,7 @@ void TestInterface::SetStaticDoubleNumber(double number) {
   TestInterface::static_double_number_ = number;
 }
 
-void TestInterface::VoidMethodTestDictionaryArg(TestDict testDict) {
+double TestInterface::VoidMethodTestDictionaryArg(TestDict testDict) {
```

Comment:
Should be `DoubleMethodTestDictionaryArg`

Reply:
done.